songs/109_Sista_punschvisan.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Sista punschvisan
 melody: Auld lang syne
+notes:
+  - Sista versen sjungs först med enda foten på stolen, sedan igen med andra foten och stolen och slutligen en sista gång under bordet.
```

Comment:
```suggestion
  - Sista versen sjungs först med ena foten på stolen, sedan igen med andra foten på stolen och slutligen en gång under bordet.
```

Fixes a couple of typos and removes the "double final" from `slutligen en sista gång`